_posts/2018-07-03-04-26-InOperator.md

```diff
@@ -0,0 +1,28 @@
+---
+title : "4.26 IN Operator"
+layout: post
+category: "4. OData features"
+---
+
+# IN Operator
+
+Starting in WebAPI OData V7.0.0 [[ASP.NET](https://www.nuget.org/packages/Microsoft.AspNet.OData/7.0.0) | [ASP.NET Core](https://www.nuget.org/packages/Microsoft.AspNetCore.OData/)], the IN operator is a [supported feature](http://docs.oasis-open.org/odata/new-in-odata/v4.01/cn01/new-in-odata-v4.01-cn01.html#_Toc485385090 "new feature") that enables a shorthand way of writing multiple EQ expressions joined by OR. For example,
```

Comment:
nit: is the "new feature" for hovering text needed?

Reply:
Thanks for the catch. It is not needed.